src/appsettings/settings.py

```diff
@@ -944,11 +1037,6 @@ def check(self):
             AttributeError: if the setting is missing and required.
             ValueError: (or other Exception) if the raw value is invalid.
         """
-        try:
-            value = self.raw_value
-        except (AttributeError, KeyError) as err:
-            self._reraise_if_required(err)
-        else:
-            self.checker(self.full_name, value)
-            for subsetting in self.settings.values():
-                subsetting.check()
+        super(NestedSetting, self).check()
+        for subsetting in self.settings.values():
+            subsetting.check()
```

Comment:
Same remark as above, should we store exception messages and concatenate them before reraising?